ui/app/routes/vault/cluster/secrets/backend/secret-edit.js

```diff
@@ -116,7 +116,7 @@ export default Route.extend({
 
   modelType(backend, secret, options = {}) {
     const backendModel = this.modelFor('vault.cluster.secrets.backend', backend);
-    const type = backendModel.get('engineType');
+    const { engineType } = backendModel;
```

Comment:
love to see all the destructuring you've added throughout this PR, it's real slick 💅